contrib/polybar.xbps/template

```diff
@@ -1,17 +1,16 @@
 # Template file for 'polybar'
 pkgname=polybar
-version=3.0.4
+version=3.1.0
 revision=1
-_i3ipcpp_version=0.6.2
-_xpp_version=1.3.6
+_i3ipcpp_version=0.7.0
+_xpp_version=1.4.0
 build_style=cmake
 configure_args="
  -DENABLE_ALSA=$(vopt_if alsa ON OFF)
  -DENABLE_CURL=$(vopt_if curl ON OFF)
  -DENABLE_I3=$(vopt_if i3 ON OFF)
  -DENABLE_MPD=$(vopt_if mpd ON OFF)
  -DENABLE_NETWORK=$(vopt_if network ON OFF)"
-depends="cairo xcb-util-wm xcb-util-image xcb-util-xrm"
```

Comment:
Does this really work w/o the dependency key?

Reply:
Yes, because in void the `-devel` packages will include the ones removed here.

Reply:
Okay great. Do they also include the new `xcb-util-cursor` dependency? I have forgotten to add this one to the readme and the wiki, I will open a PR for it shortly.

Reply:
No. That is not included.